### PR TITLE
feat: If wallet empty = bigger button for Add Funds

### LINKED ISSUE #8419: If wallet empty = bigger button for Add Funds
Just a small design request, nothing urgent. I noticed that when someone has empty wallet (no funds) that the button add funds is very subtle, and small.

Can we make the button **bigger** (proppers nice big visible button) in case the wallet is empty? So the person sees it right away and its more convenient and confident to click on it.

Screenshot 2023-12-05 at 13 58 35

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

  👇 __ Let's make a quick check before the contribution.

  ## PR Type

  - [x] Feature

  ## Needs Design check

  - @exezbcz please review

  ## Context

  - [x] Closes #8419

  #### Did your issue had any of the "$" label on it?

  - [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)

  ## Screenshot 📸

  - [x] My fix has changed UI

image

  ## Copilot Summary
  copilot:summary

  copilot:poem

### CODE REVIEW

File: components/balance/MultipleBalances.vue [hunk 1]
```diff
@@ -2,8 +2,17 @@
   <div class="mb-2">
     <div
       v-if="isEmptyBalanceOnAllChains && !isBalanceLoading"
-      class="has-text-grey">
-      {{ $t('asset.emptyAsset') }}
+      class="is-size-7 py-4 flex flex-col items-center">
+      <div class="mb-3 text-align-center">
```
Comment (roiLeo) [inline components/balance/MultipleBalances.vue:6]:
```suggestion
      <div class="mb-3 text-center">
```

File: components/balance/MultipleBalances.vue [hunk 2]
```diff
@@ -2,8 +2,17 @@
   <div class="mb-2">
     <div
       v-if="isEmptyBalanceOnAllChains && !isBalanceLoading"
-      class="has-text-grey">
-      {{ $t('asset.emptyAsset') }}
+      class="is-size-7 py-4 flex flex-col items-center">
```
Comment (roiLeo) [inline components/balance/MultipleBalances.vue:5]:
```suggestion
      class="text-xs py-4 flex flex-col items-center">
```

File: components/balance/MultipleBalances.vue [hunk 3]
```diff
@@ -55,13 +64,23 @@
       >
     </p>
   </div>
+
+  <div
+    v-if="!isEmptyBalanceOnAllChains && !isBalanceLoading"
+    class="flex items-center justify-between">
+    <a class="has-text-grey is-size-7" @click="openRampModal"
```
Comment (roiLeo) [inline components/balance/MultipleBalances.vue:71]:
```suggestion
    <a class="text-grey text-xs" @click="openRampModal"
```

**Review by roiLeo — APPROVED** [review]

**Comment by yangwao** [comment]
Thanks!
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 8.29 USD/DOT ~ 3.619 $DOT
🧗 [16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z](https://polkadot.subscan.io/account/16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)
🔗 [0x819de8e1771ec3db6d8e66d17805991022733cb1336bf96a234755aba8d637db](https://polkadot.subscan.io/extrinsic/0x819de8e1771ec3db6d8e66d17805991022733cb1336bf96a234755aba8d637db)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**